exporter/loadbalancingexporter/README.md

```diff
@@ -63,6 +63,12 @@ Refer to [config.yaml](./testdata/config.yaml) for detailed examples on using th
 * The `k8s` node accepts the following optional properties:
   * `service` Kubernetes service to resolve, e.g. `lb-svc.lb-ns`. If no namespace is specified, an attempt will be made to infer the namespace for this collector, and if this fails it will fall back to the `default` namespace.
   * `ports` port to be used for exporting the traces to the addresses resolved from `service`. If `ports` is not specified, the default port 4317 is used. When multiple ports are specified, two backends are added to the load balancer as if they were at different pods.
+* The `awsCloudMap` node accepts the following properties:
+  * `namespace` The CloudMap namespace where the service is register, e.g. `cloudmap`. If no `namespace` is specified, this will fail to start the Load Balancer exporter.
+  * `serviceName` The name of the service that you specified when you registered the instance, e.g. `otelcollectors`.  If no `serviceName` is specified, this will fail to start the Load Balancer exporter.
+  * `interval` resolver interval in go-Duration format, e.g. `5s`, `1d`, `30m`. If not specified, `30s` will be used.
+  * `timeout` resolver timeout in go-Duration format, e.g. `5s`, `1d`, `30m`. If not specified, `1s` will be used.
+  * `port` port to be used for exporting the traces to the addresses resolved from `service`. By default, the port is set in Cloud Map, but can be be overridden with a static value in this config
```

Comment:
There seems to be a `healthStatus` property as well. Should it be documented?

Reply:
For this property, in this initial iteration, I'd rather to leave this config as default since we filter only for Healthy instances

Reference: https://docs.aws.amazon.com/cloud-map/latest/api/API_GetInstancesHealthStatus.html

Reply:
I might be missing something, but the value the user is providing is indeed being used, and therefore, deserves to be documented.